### ISSUE TITLE
[iOS] Conditional GET requests do not work

### ISSUE THREAD

**Issue by jkrause** [eclipsesource/tabris-js#697]
Conditional get requests throw an error under iOS instead providing a response with status code 304 (not modified). Android works fine.

To reproduce run this code twice. The first time it should report status code 200, the second time status code 304 (not modified).

```
var lastModified = localStorage.getItem("lastModified");

var xhr = new XMLHttpRequest();
xhr.open('GET', "http://developer.eclipsesource.com/");
if(lastModified) {
  xhr.setRequestHeader("If-Modified-Since", lastModified);
}
xhr.onreadystatechange = function() {
  if (this.readyState == this.DONE) {
    console.log("status", xhr.status);
    if(xhr.status === 200) {
      localStorage.setItem("lastModified", xhr.getResponseHeader("Last-Modified"));      
    }
  }
};
xhr.onerror = function(err) {
  if (this.readyState == this.DONE) {
    console.log("error", err);
  }
};

xhr.send();
```

**Comment by jkrause** [comment]
See also https://github.com/AFNetworking/AFNetworking/pull/928

**Comment by patrykmol** [comment]
Fixed for 1.5 release.
